samples/FAControlsGallery/ViewModels/SettingsPageViewModel.cs

```diff
@@ -10,30 +10,58 @@ namespace FAControlsGallery.ViewModels;
 
 public class SettingsPageViewModel : MainPageViewModelBase
 {
+    readonly FluentAvaloniaTheme faTheme;
```

Comment:
minor: `private readonly FluentAvaloniaTheme _faTheme;`, and move it to the bottom with the other fields

Reply:
ok, done.